test/api/sources.ts

```diff
@@ -57,8 +58,12 @@ test('Source events', async t => {
   event = await client.fetchNextEvent();
   t.is(event.data.name, 'audio2');
 
-  // check `sourceUpdated` event when renaming a source
-  source1.setName('audio3');
-  event = await client.fetchNextEvent();
-  t.is(event.data.name, 'audio3');
+  // // check `sourceUpdated` event when renaming a source
+  // source1.setName('audio3');
+  // event = await client.fetchNextEvent();
+  //
+  // // the remote control app requires these fields to be in the event
+  // t.is(event.data.name, 'audio3');
+  // t.is(event.data.configurable, true);
+  // t.truthy(event.data.resourceId);
```

Comment:
Is there a reason we're keeping these but commented out?  Seems like this is supposed to be testing for the regression you just fixed.

Reply:
Didn't push the last commit. Good catch